src/clients/AcrossConfigStoreClient/AcrossConfigStoreClient.ts

```diff
@@ -123,6 +124,13 @@ export class AcrossConfigStoreClient extends BaseAbstractClient {
     return across.rateModel.parseAndReturnRateModelFromString(config.routeRateModel[route]);
   }
 
+  getChainIdIndicesForBlock(blockNumber: number = Number.MAX_SAFE_INTEGER): number[] {
+    const config = (sortEventsDescending(this.chainIdIndicesUpdates) as GlobalConfigUpdate<number[]>[]).find(
+      (config) => config.blockNumber <= blockNumber
+    );
+    return config?.value ?? [1, 10, 137, 288, 42161];
```

Comment:
nit: can you add a comment how you arrive at this default value? i.e. some link to the UMIP ?

Reply:
Done: https://github.com/across-protocol/sdk-v2/pull/346/commits/b2b3a6f73d5b7bf174df83112bb32b0d85b3bae8